nodejs/week1/avg.js

```diff
@@ -0,0 +1,20 @@
+// calculate the average of the numbers provided as command line arguments.
+const args = process.argv.slice(2);
+const numbers = args.map(Number);
+// Check if all arguments are numbers
+if (numbers.some(isNaN)) {
+  console.error("Please provide valid numbers.");
+  process.exit(1);
+}
+// Check if the array is empty
+if (numbers.length === 0) {
+  console.error("Please provide at least one number.");
+  process.exit(1);
```

Comment:
I would suggest to keep length check logic once you get args Array so in worst case code flow will stop if no input .